redirects/kmm.yml

```diff
@@ -20,9 +20,6 @@
 - from: /docs/mobile/discover-kmm-project.html
   to: /docs/multiplatform-mobile-understand-project-structure.html
 
-- from: /docs/mobile/introduce-your-team-to-kmm.html
```

Comment:
But now there is no redirect from "/docs/mobile/introduce-your-team-to-kmm.html" to anywhere

Reply:
https://github.com/JetBrains/kotlin-web-site/pull/4145 fixed here